backbone/detect.py

```diff
@@ -5,11 +5,8 @@
 class Detector():
     def __init__(self):
         self.model = torch.hub.load('ultralytics/yolov5', 'custom',
-                               path=os.environ.get('MODEL_PATH', 'weights/best.pt'))
-    def set_model(self, model_path):
-        self.model = torch.hub.load('ultralytics/yolov5', 'custom',
-                               path=model_path)
-    
+                                    path=os.environ.get('MODEL_PATH', 'weights/best.pt'))
```

Comment:
why do change it? it should get a dynamic path. maybe someone want to change the weight, Its not rational

Reply:
yes. I forget to paste it when have test fastapi version.